packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -390,7 +390,7 @@ class Opacity extends SingleChildRenderObjectWidget {
 ///     ).createShader(bounds);
 ///   },
 ///   child: const Text(
-///     'I’m burning the memories',
+///     'I'm burning the memories',
```

Comment:
If you want to use `'` in a string you have to either escape it like `\'` or use `"` as a string delimiter:
```suggestion
///     "I'm burning the memories",
```